lightning/src/util/config.rs

```diff
@@ -206,7 +206,20 @@ pub struct ChannelConfig {
 	/// This cannot be changed after a channel has been initialized.
 	///
 	/// Default value: true.
-	pub commit_upfront_shutdown_pubkey: bool
+	pub commit_upfront_shutdown_pubkey: bool,
+	/// Limit holder balance exposure to dusted HTLCs at any given time.
+	///
+	/// Applied on at HTLC reception/forwarding, *each* on both-sides commitment transactions.
+	///
+	/// Dust HTLCs are defined as any HTLC lower than counterparty's `dust_limit_satoshis` or
+	/// holder's `dust_limit_satoshis` summed up with their expected onchain claim cost at
+	/// negotiated feerate (`feerate_per_kw` * HTLC_*_TX_WEIGHT * 1000  / 1000).
+	///
+	/// This limit does exist to prevent holder balance burnt to miners fees if the link is
+	/// broken, either due to a accidental force-closure or a counterparty maliciously
+	/// cooperating with miners.
```

Comment:
This includes way too much technical detail about the implementation and not enough detail about what this *means* for users.

```suggestion
	/// Limit our total exposure to in-flight HTLCs which are burned to fees as they are too small.
	/// to claim on-chain.
	///
	/// When an HTLC present in one of our channels is below a "dust" threshold, the HTLC will not
	/// be claimable on-chain, instead being turned into additional miner fees if either party
	/// force-closes the channel. Because the threshold is per-HTLC, our total exposure to such
	/// payments may be substantial if there are many dust HTLCs present when the channel is
	/// force-closed.
	///
	/// This limit is applied for send, forwarded, and received HTLCs and limits the total exposure
	/// across all three types per-channel. Setting this too low may prevent the sending or receipt
	/// of low-value HTLCs on high-traffic nodes, and is very important if your counterparty is a
	/// miner (as they could benefit from such dust HTLCs becoming miner fees).
```

Reply:
Better, though I would recall the last part about counterparty=miner being a worrying concern is kind of helpless. Even assuming you can effectively identify your LN counterparty, you can't exclude unseen out-of-band negotiation with a miner pool to introduce such non-propagated "dust-stealing tx".

Reply:
Yea, I think you're right. It would give users too much of a false sense of security.

Reply:
Yep modified a bit.